modules/management-vpc/README.md

```diff
@@ -6,6 +6,16 @@ You can use this submodule when you need more modularity to create your topology
 
 See the [Landing Zone example](../../examples/landing_zone/) for runnable code.
 
+### Usage
+```terraform
+module management_vpc {
+  source              = https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/tree/main/modules/management-vpc
```

Comment:
thats not how you consume a submodule from github, it would be: "git::https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc.git//modules/management-vpc?ref=X.X.X"

However, lets use hashicorp as the source, which would be:
```
 source                      = "terraform-ibm-modules/terraform-ibm-landing-zone-vpc/ibm//modules/management-vpc"
version                     = "X.X.X" # replace X.X.X with a release version to lock into an exact release
```